code/gateway/kxdash.q

```diff
@@ -57,5 +52,5 @@ init:{
  // incorporate dashps into the .z.ps definition
  .dotz.set[`.z.ps;{x@y;.kxdash.dashps y}@[value;.dotz.getcommand[`.z.ps];{{value x}}]];
```

Comment:
`.kxdash.dashps` is executed after the original `.z.ps` definition - are we actually using `dashparams` anywhere after this change? or the `dashremote` and `dashjoin` functions? Seems like most of this script is possibly redundant if we're just using the original `.z.ps`?